src/main/java/com/danielfrak/code/keycloak/providers/rest/ConfigurationProperties.java

```diff
@@ -23,6 +24,10 @@
             new ProviderConfigProperty(API_TOKEN_PROPERTY,
                     "Rest client API token", "Bearer token",
                     PASSWORD, null),
+            new ProviderConfigProperty(USE_USER_ID_FOR_CREDENTIAL_VERIFICATION,
+                    "Use user id for credential verification",
+                    "Use the id of the user as the path parameter when making a credential verification request",
```

Comment:
What do you think about adding an `instead of the username` to the description here?